docs/en/solutions/How_to_Migrate_CNI_from_Calico_to_Kube-OVN_on_ACP.md

```
@@ -0,0 +1,233 @@
+
+### Step 7: Final Verification
+
+**Operate on the business cluster**
+
+```bash
+kubectl get apprelease -n cpaas-system | grep cni-
+```
+
+Expected output:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)